src/Raven.Server/Documents/Indexes/Static/StaticIndexBase.cs

```diff
@@ -56,7 +56,7 @@ public dynamic LoadAttachments(dynamic doc)
             if (document == null)
                 throw new InvalidOperationException($"{nameof(LoadAttachments)} may only be called with a non-null entity as a parameter, but was called with a parameter of type {doc.GetType().FullName}: {doc}");
 
-            return CurrentIndexingScope.Current.LoadAttachments(document);
+            return new DynamicArray(CurrentIndexingScope.Current.LoadAttachments(document));
```

Comment:
CurrentIndexingScope.LoadAttachments returns dynamic, I think then we need to move this change there and remove it from here? @arekpalinski ?

Reply:
Yes, I think it would be slightly safer. @Lwiel  let's try to move this change there.

Reply:
Done